src/core/src/bootstrap/Constants.py

```diff
@@ -98,6 +98,33 @@ class ConfigSettings(EnumBackport):
     # patch versions for healthstore when there is no maintenance run id
     PATCH_VERSION_UNKNOWN = "UNKNOWN"
 
+    # Strings used in perf logs
+    class LogStrings:
+        TASK = "task"
+        TASK_STATUS = "task_status"
+        ASSESSMENT = "Assessment"
+        PATCH_INSTALLATION = "PatchInstallation"
+        PACKAGE_MANAGER = "package_manager"
+        NUMBER_OF_TRIALS = "number_of_trials"
+        NUMBER_OF_ROUNDS = "number_of_rounds"
```

Comment:
What is the difference between NUMBER_OF_TRIALS and NUMBER_OF_ROUNDS?

Reply:
NUMBER_OF_TRIALS: Used in assessment. If there are any failures during assessment, the assessment is tried again. So it is number of tried to do asessment.

NUMBER_OF_ROUNDS: Used in install patches. Based on the flag PACKAGE_MGR_SETTING_REPEAT_PATCH_OPERATION (which is used in Zypper only), we do more than one round of install patching in some scenarios.